components/server/src/billing/entitlement-service.ts

```diff
@@ -182,13 +182,13 @@ export class EntitlementServiceImpl implements EntitlementService {
             const billingMode = await this.billingModes.getBillingMode(userId, organizationId);
             switch (billingMode.mode) {
                 case "none":
-                    return false;
+                    return true;
```

Comment:
@kylos101 Is that really what we want? We will not limit network connections for all Dedicated cells...?

Reply:
An alternative would be to enable UBP for integration test users in our preview environments. Any idea how we might do that?

Re: this change as it:
My recollection is, this should be fine, because limiting kicks in at a really high rate to protect the node and other workspaces.

Reply:
@geropl fixed in 643701c6352609b3bab3090d271d38cb42b33b3c and c741a705ce4faf55d233392343fb9e4ae3adb680.

Can you think of why were not enabling UBP by default for preview environments? 🤔

Reply:
Whu, really no clue where the stripeKeys line went before :open_mouth: I'm like 99.99% certain we had it before... We should be able to turn it off (for a more cell-like behavior), but IMO that's `GITPOD_WITH_DEDICATED_EMU` is for, so: LGTM! :heavy_check_mark: